tor/core/inbox.py

```diff
@@ -86,7 +86,7 @@ def process_reply(reply, config):
             reply.mark_read()
             return
 
-        if 'claim' in r_body:
+        if 'claim' in r_body or 'dibs' in r_body:
```

Comment:
For stylistic purposes, I'd like to keep the parentheses like with the `done` command. Makes it easier to see that there are two separate commands which trigger the same functions.